### PR TITLE
feat:changed name of button to 'Save School'

### LINKED ISSUE #36: Name Change
### Description

In the add section the text in the signin button should be "add" and not "signin"
and once the button is clicked the school should appear in the schools field

screenshot of the issue can be seen below
Screenshot from 2022-08-02 13-19-46

### PR DESCRIPTION
**Pull request by mofirojean**
Changed the name of the button from signin to Save School

### Contributor Checklist

Fixes #36 

- [ ] Did you add comments for every public function/method?
- [x] Did you specify the type of improvement?
- [x] Did you request a desired merge strategy?

#### Type of Improvement

- Simple Issue

#### Backend Code Generation Impact

- NA

#### Desired Merge Strategy

- Squash and Merge

#### Release Notes

- NA

### Reviewer Checklist (only modified by reviewer)
- [x] Did you add the appropriate labels?
- [x] Did you review?
- [x] Did you check whether all relevant Contributor checkboxes have been checked?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -35,7 +36,7 @@
   },
   "devDependencies": {
     "@angular-devkit/build-angular": "~13.2.6",
-    "@angular/cli": "~13.2.6",
+    "@angular/cli": "^13.2.6",
```
Comment (elroykanye) [inline package.json:39]: This too

File: package.json [hunk 2]
```diff
@@ -24,6 +24,7 @@
     "@popperjs/core": "^2.10.2",
     "bootstrap": "^5.1.3",
     "bootstrap-icons": "^1.8.2",
+    "g": "^2.0.1",
```
Comment (elroykanye) [inline package.json:27]: I don't get the need for this change as it does not have anything to do with this PR

**Review by elroykanye — CHANGES_REQUESTED** [review]
Thanks!

Undo these changes that do not relate to the PR :)

File: src/app/components/select-school/select-school.component.html [hunk 3]
```diff
@@ -45,7 +45,7 @@
               <label for="open" class="text-900 font-medium">Applications Open</label>
             </div>
 
-            <button (click)="addSchoolAction()" pButton pRipple label="Sign In" icon="pi pi-user" class="w-full"></button>
+            <button (click)="addSchoolAction()" pButton pRipple label="Save School" class="w-full"></button>
```
Comment (elroykanye) [inline src/app/components/select-school/select-school.component.html:48]: You could consider changing the icon instead.

Comment (mofirojean) [reply src/app/components/select-school/select-school.component.html:48]: ok thanks

Comment (elroykanye) [reply src/app/components/select-school/select-school.component.html:48]: https://www.primefaces.org/primeng/icons

Refer to this

**Review by elroykanye — APPROVED** [review]

**Pull request merged**